core/src/main/java/org/springframework/ldap/query/ConditionCriteria.java

```diff
@@ -105,4 +105,15 @@ public interface ConditionCriteria {
 	 */
 	ConditionCriteria not();
 
+	/**
+	 * Appends an {@link org.springframework.ldap.filter.ProximityFilter}.
+	 * @param value the value to compare with.
+	 * @return an ContainerCriteria instance that can be used to continue append more
+	 * criteria or as the LdapQuery instance to be used as instance to e.g.
+	 * {@link org.springframework.ldap.core.LdapOperations#search(LdapQuery, org.springframework.ldap.core.ContextMapper)}.
+	 *
+	 * @see org.springframework.ldap.filter.EqualsFilter
+	 */
+	ContainerCriteria proximity(String value);
```

Comment:
This needs to have a `default` implementation to remain backward compatible. The default implementation would need to throw an `UnsupportedOperationException`